scripts/zones/RoMaeve/IDs.lua

```diff
@@ -53,6 +53,15 @@ zones[dsp.zone.ROMAEVE] =
         },
         MOONGATE_OFFSET = 17277195,
         BASTOK_7_1_QM   = 17277207,
+        MOONGATE_PASS_QM_POS =
+        {
+            {-277.651,-3.765, -17.895},
+            { 264.681, 4.000, -52.630},
+            { 278.402, 4.993,  -3.200},
+            { 151.779, 4.719,  68.553},
+            {-134.518, 4.000, 106.042},
+        },
```

Comment:
Do we usually put Pos in these NPC ID tables? Would they not go in the NPC script?

Reply:
@ibm2431  I have seen them for other mobs, but let me know if I should put it back inside 😸

Reply:
@ibm2431  in the same file bastok quest QM also stores it there.

Reply:
Since they are not "IDs" at all, I think those should eventually be moved to the zone helper functions
https://github.com/DarkstarProject/darkstar/blob/master/scripts/zones/PsoXja/globals.lua
like that. We use those as sort of zone specific globals - its not game wide, but it is intended for stuff used by multiple things in the zone.

For the purpose of this PR..Currently exactly 1 script will use those. They may as well be in that one script as a local table instead. Alternatively, leave them where they are but also set the position at server startup in the zone.lua. If in the zone lua, you _will_ need to "GetByID". In the NPC's own script, you _will not_ need to do that.

Reply:
tl;dr don't global anything that does not need to be global.

Reply:
that makes total sense, since this is only used by one NPC it doesn't need to be global. Thank @TeoTwawki  making the changes now 😸